fastly/resource_fastly_service_wasm_v1.go

```diff
@@ -0,0 +1,23 @@
+package fastly
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+// Ordering is important - stored is processing order
+// Conditions need to be updated first, as they can be referenced by other
+// configuration objects (Backends, Request Headers, etc)
+
```

Comment:
While true for VCL services, objects such as conditions don't exist on Wasm services, so this should be reworded. 
```suggestion
// Ordering is important - stored is processing order
// Some objects may need to be updated first, as they can be referenced by other
// configuration objects (Backends, Request Headers, etc).
```